modules/unsupported/process-feature/src/main/java/org/geotools/process/vector/PointStackerProcess.java

```diff
@@ -395,19 +573,34 @@ private SimpleFeatureType createType(CoordinateReferenceSystem crs, boolean stre
         tb.add(ATTR_GEOM, Point.class, crs);
         tb.add(ATTR_COUNT, Integer.class);
         tb.add(ATTR_COUNT_UNIQUE, Integer.class);
-        tb.add(ATTR_BOUNDING_BOX_GEOM, Polygon.class);
-        tb.add(ATTR_BOUNDING_BOX, String.class);
+        tb.add(ATTR_BOUNDING_AREA, Geometry.class);
+        tb.add(ATTR_BOUNDING_BOX, List.class);
+
         if (stretch) {
             tb.add(ATTR_NORM_COUNT, Double.class);
             tb.add(ATTR_NORM_COUNT_UNIQUE, Double.class);
         }
+        tb.add(ATTR_CLUSTERED_VALUES, JSONArray.class);
```

Comment:
As previously said I found that JSONArray will be automatically converted to json  (using rest, although I found 2 diffrent usecases at the geoserver level: the geoserver 2.8  field is converted in a embedded json (which had to be parsed independently) and in geoserver 2.15 it is converted as part of the entire rest resppnse. Given the two I thought JSONArray  is a safe bet related to compatibility so I integrated wfs calls in a JS library to handle those 2 cases. So what should be course of action?

Reply:
The first bit is to verify what actually breaks the test down in GeoServer.

Reply:
I will try recompile everything locally, but I feel it will a long task (regarding geoserver, seems that i have weird issues related to some resources..). My first impression that that was related to some PMD issue and an uninitialized variable, I later, corrected, the PointStacker failed silently and the test went bad... but we will see.

Reply:
I found out the the issue of failing was related to jts.version which in geoserver is linked to 1.16.1 but in geotools is 1.17.1, and the failing method was getExteriorRing. Now, I really do not understand why this happened... but syncing versions allowed the test `geoserver\src\wms>mvn clean install -Dtest=org.geoserver.wms.wms_1_1_1.GetFeatureInfoTest#testPointStacker` to run.